### DISCUSSION TITLE
Vertical Slice Architecture in Oqtane Module

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#3286] [General] [answered]
Hi,

I wanted to create features for each entities (organization, division, department, section & unit) using Vertical Slice Architecture (VSA). How do I do that in Oqtane Module?

**Comment by sbwalker** [comment, marked as answer]
An Oqtane module is a vertical slice as it encapsulates both the client-side UI and back-end services/data for a feature and allows you to package and deploy the feature independently from the core framework.

**Reply by Sharafudeen** [reply]
Within a module can manage multiple entities UI and deploy as a package for each entity with different versions of each entities?
